packages/intltool/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://launchpad.net/intltool
+TERMUX_PKG_DESCRIPTION="Automatically extracts translatable strings from oaf, glade, bonobo ui, nautilus theme and other XML files into the po files."
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_VERSION=0.51.0
+TERMUX_PKG_DEPENDS="perl"
+TERMUX_PKG_SRCURL=https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--prefix=$TERMUX_PREFIX"
+termux_step_pre_configure() {
+cpan install XML::Parser
```

Comment:
This installs xml::parser in the docker image. Instead of installing it in the docker image through cpan you should install it through apt by adding it to [setup-ubuntu.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh). 

xml::parser needs to be installed on device for the package to work. Only way to install it on device would be in a postinst script.

Reply:
should I add 
```
PACKAGES+="  xml::parser"
```
to build-ubuntu.sh?

Reply:
No, there's no ubuntu package called xml::parser so that would not work. It is called [libxml-parser-perl](https://launchpad.net/ubuntu/xenial/+package/libxml-parser-perl)

Reply:
Got it